test/mock/mock-api.js

```diff
@@ -32,11 +32,11 @@ define([
     });
 
     function noop() {
-        //console.log('I shouldn\'t exist.');
+        // console.log('I shouldn\'t exist.');
```

Comment:
Yeah, as we transition from `assert` to `expect`, I plan to do a significant amount of cleanup of the unit tests codes. Already did a bunch of linting.